Makefile

```diff
@@ -1,24 +1,46 @@
-.PHONY: quality_checks quality style fixup test test-examples docs
+.PHONY: modified_only_fixup extra_quality_checks quality style fixup fix-copies test test-examples docs
+
+
+check_dirs := examples templates tests src utils
```

Comment:
This has always needed to be run from the repo root, right?

Reply:
Yes, since `Makefile` is in root of the repo.

Reply:
I haven't changed any behavior, just replaced a multiple duplicated listings with a single variable.